### ISSUE TITLE
BIOS SVM enabled causes driver to fail with driver code 43 and hyper-V event 16000

### ISSUE THREAD

**Issue by jschoch** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#177]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Windows

### Processor / Processor Number [Required]

Ryzen 3700x

### Graphic Card [Required]

a770

### GPU Driver Version [Required]

- [X] 31.0.101.3793 (*latest*)
- [ ] 31.0.101.3491
- [ ] 31.0.101.3490
- [ ] 31.0.101.3430
- [ ] 31.0.101.3276
- [ ] 30.0.101.3268
- [ ] 30.0.101.3259
- [ ] 30.0.101.1743

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H1 (*October 2021 Update*)
- [ ] Windows 10 21H2 (*November 2021 Update*)
- [ ] Windows 10 21H1 (*May 2021 Update*)
- [ ] Windows 10 20H2 (*October 2020 Update*)
- [ ] Other (Please specify)

### Intel System Support Utility report

no time

### Description and steps to reproduce [Required]

in my bios for my asus tuf x570 pro wifi I enable SVM (secure virtual machine) and the PC will boot and post but nothing will appear on the screen.  RDP'ing into the machine and inspecting device manager reveals hyper-v has issues and the A770 driver is disabled which is why there is nothing displaying on the screen.  Disabling SRV in the BIOS resolves the problem.

**Comment by IntelSupport-Rozilah** [comment]
hi @jschoch,
Thanks for contacting Intel, could you please attach the SSU detail for us to verify the issue?

**Comment by IntelSupport-Rozilah** [comment]
hi @jschoch,
could you please attach the SSU details?

**Comment by IntelSupport-Rozilah** [comment]
hi @jschoch,

i have verify the issue, the issue is not reproducible on our system and no yellow bang or error after enable/disable SVM, i able to get the display working from A770 as below:
1. Disable SVM
![SVM disable](https://user-images.githubusercontent.com/44628772/202391725-0168ca29-5197-408d-84c9-097a284b84a2.png)
2. Enable SVM
![SVM Enable](https://user-images.githubusercontent.com/44628772/202391788-5f20c203-71d9-4ffd-bd7b-031758f5d9d1.png)

could you try change the display cable from HDMI to DP if currently you use HDMI into other port of A770?

**Comment by jschoch** [comment]
Both cables were DP.

There is no "yellow bang", when the machine tries to load windows the
driver crashes and the result is the user just see's a "black screen".  It
didn't always happen immediately, sometimes took a second or two after the
windows login screen appeared.

I sold the card so I can't test any more.

**Comment by IntelSupport-Rozilah** [comment]
hi @jschoch 
I would like to suggest you update the bios if your current bios are not up to date and test with the latest driver version [31.0.101.3802](https://www.intel.com/content/www/us/en/download/729157/intel-arc-graphics-windows-dch-driver-beta.html) but I'm sorry to hear that you sold the card and couldn't test the issue any longer.
Since this issue was not observed on multiple platforms in our lab using the most recent driver, we would like to close this case. Our advice would be for you to contact your device manufacturer support since the issue seems to be specific to your device. In the future, if you need assistance with this issue or any other issue, please raise it and we will be happy to assist.

**Comment by IntelSupport-Arun** [comment]
@IGCIT , According to our lab testing, the issue has not been observed across multiple platforms using the latest drivers. The customer was not able to do further verification as the card was not available to them. Please proceed to close this case. Thanks

**Comment by daffodilistic** [comment]
Hi I have a similar issue, should I create a new issue or post up my bug/error report in this issue?

**Comment by IGCIT** [comment]
if it is a different error, open a new one, otherwise just add your feedback here and i will reopen

**Comment by afranche** [comment]
I am experiencing the same similar issue with an A770 on driver version 31.0.101.4255. After enabling SVM on my motherboard with the driver installed, screen does not start and I experience the same exact problem as mentioned by the issue owner.

I had to reboot multiple times to trigger safe mode and uninstall the driver from there. Screen was working again.

Specs:
- Gigabyte A520I AC
- AMD Ryzen 5 2600
- Arc A770 LE

I can provide screenshots and/or more information if required.

**Comment by IGCIT** [comment]
@afranche yes, more info, and maybe a video too, could help intel to identify the issue

**Comment by afranche** [comment]
Sure thing. I'll just dump a quick HWiNFO screenshot since this is the quickest I can do for now. It's quite late on CET time so I'll get back tomorrow with a video where I'll reproduce the case. 👍🏼 

![image](https://user-images.githubusercontent.com/11685232/230745337-11ab68b5-ffbd-419d-8c85-7c3c354e5c83.png)

**Comment by afranche** [comment]
@IGCIT So coming back here after a while to sort out some issues on my side, I've finally got my hands on a recording device to properly show what's happening, from start to finish. https://youtu.be/NFOAeZJpfcY

Here are the screenshots from the Intel ARC Control configuration panels just in case:

![General information](https://user-images.githubusercontent.com/11685232/235327466-9114a21b-11b8-4113-8c57-9f06b36a9370.png)
![GPU information](https://user-images.githubusercontent.com/11685232/235327468-5351d6b1-dec0-43ed-9ba8-9f0f870efee4.png)
![Software version](https://user-images.githubusercontent.com/11685232/235327476-4ab2dbb8-928b-4d7e-abb5-6065a2134c57.png)
![System capabilities](https://user-images.githubusercontent.com/11685232/235327481-c39c8612-15d9-4cd0-95ab-482ffa09e90c.png)

Sorry for the delay. I've also set up RDP so if more infos are needed while the issue is happening, let me know. 🙏🏼

**Comment by SavageWaffle** [comment]
@Felipe-Intel Is there any update regarding this?

**Comment by Felipe-Intel** [comment]
Internal ticket was closed. If anyone is still experiencing this problem, please open a new ticket.
